src/index.tsx

```diff
@@ -1,2 +1,4 @@
 export * from './functions'
```

Comment:
functions should not be exported anymore.

Reply:
we have to do this one later on as implementing central still leaves peripheral to be implemented which break the example app this way I think

Reply:
Thats fine, we are fixing both at the same time.

Reply:
alright I'll remove it then